src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -218,7 +220,7 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   // Local properties
   authorization = null as ContinuationAuthorizationIF
   fileValidity = false
-  fileUploadCustomErrorMsg = ''
+  customErrorMessage = ['', '', '', '', '']
```

Comment:
A cool way would also be `Array(5).fill('')` 😆 

Up to you Sev. No need to change anything.

Reply:
Good point. I could also use a constant for the number of allowed file uploads. If I'm in there again I might update it :)